Source/QED/breit_wheeler_engine_wrapper.cpp

```diff
@@ -0,0 +1,16 @@
+#include "breit_wheeler_engine_wrapper.h"
+//This file provides a wrapper aroud the breit_wheeler engine
+//provided by the standard template library
+
+using namespace picsar::multi_physics;
+
+warpx_breit_wheeler_engine::warpx_breit_wheeler_engine():
+    breit_wheeler_engine(std::move(amrex_rng_wrapper{}))
+{};
+
+//Interface for the get_optical_depth method of the BW engine
+amrex::Real
+AMREX_GPU_HOST_DEVICE
+warpx_breit_wheeler_engine::get_optical_depth(){
+    return internal_get_optical_depth(amrex::Random());
+}
```

Comment:
this (static) function is a wrapper around the static function internal_get_optical_depth of the base class.
We might want to change the name, because get_optical_depth() is also the name of a non static member function of the base class. I could replace it with "optical_depth".
Actually internal_get_optical_depth does a very simple thing:

```
template<typename _REAL, class _RNDWRAP>
PXRMP_GPU
PXRMP_FORCE_INLINE
_REAL picsar::multi_physics::breit_wheeler_engine<_REAL, _RNDWRAP>::
internal_get_optical_depth(_REAL unf_zero_one_minus_epsi)
{
    return -log(one - unf_zero_one_minus_epsi);
}
```

Reply:
thanks, I'll keep it as it is for the moment